go/gosec/bad_crypto/use_of_weak_crypto.yaml

```diff
@@ -0,0 +1,59 @@
+rules:
+  - id: use_of_md5
+    message:
+      Use of weak cryptographic primitive MD5
+    languages: [go]
+    severity: WARNING
+    metadata:
+      owasp: 'A9: Using Components with Known Vulnerabilities'
+      cwe: 'CWE-327: Use of a Broken or Risky Cryptographic Algorithm'
+      source_rule_url: https://github.com/securego/gosec#available-rules
+    patterns:
```

Comment:
`pattern-either` can now be used as a top-level key, so we don't need to nest it underneath `patterns` anymore. Same to the below usages.

Reply:
ok